packages/circuit-ui/components/Avatar/Avatar.tsx

```diff
@@ -55,45 +61,63 @@ const placeholders = {
     '<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 96 96"><path fill="white" d="M48 18c-7.732 0-14 6.268-14 14s6.268 14 14 14 14-6.268 14-14-6.268-14-14-14zM47.9998 88C61.53 88 73.4913 81.2822 80.73 71c-7.2387-10.2822-19.2-17-32.7303-17-13.5302 0-25.4914 6.7178-32.7302 17 7.2388 10.2822 19.2 17 32.7303 17z"/></svg>',
 };
 
-type StyledImageProps = Omit<AvatarProps, 'size' | 'variant'> & {
+type StyledProps = {
   size: AvatarSize;
   variant: AvatarVariant;
 };
 
-const baseStyles = ({ theme, size }: StyledImageProps & StyleProps) => css`
-  display: block;
+const baseStyles = ({ theme, size }: StyledProps & StyleProps) => css`
   width: ${avatarSizes[size]};
   height: ${avatarSizes[size]};
   box-shadow: 0 0 0 ${theme.borderWidth.kilo} rgba(0, 0, 0, 0.1);
-
   background-color: ${theme.colors.n300};
+`;
+
+const imageStyles = () => css`
+  display: block;
   object-fit: cover;
   object-position: center;
 `;
 
-const borderStyles = ({
+const borderRadiusStyles = ({
   theme,
   variant,
   size,
-}: StyledImageProps & StyleProps) => {
-  let styles = css`
-    border-radius: ${theme.borderRadius.kilo};
-  `;
+}: StyledProps & StyleProps) => {
   if (variant === 'identity') {
-    styles = css`
+    return css`
       border-radius: ${theme.borderRadius.circle};
     `;
-  } else if (size === 'giga') {
-    styles = css`
+  }
+
+  if (size === 'giga') {
+    return css`
       border-radius: ${theme.borderRadius.byte};
     `;
   }
-  return styles;
+
+  return css`
+    border-radius: ${theme.borderRadius.kilo};
+  `;
 };
 
-const StyledImage = styled('img', {
+const Image = styled('img', {
+  shouldForwardProp: (prop) => isPropValid(prop),
+})<StyledProps>(baseStyles, borderRadiusStyles, imageStyles);
+
+const initialStyles = ({ theme, size }: StyledProps & StyleProps) => css`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: ${theme.colors.white};
+  font-size: calc(${avatarSizes[size]} / 2);
+  font-weight: ${theme.fontWeight.bold};
+  user-select: none;
```

Comment:
I think a user should not be able to select the initials since they are presentational only. 

Let's say a user wants to copy a list of names. If they are displayed alongside the `Avatar` component with initials, it would be annoying if the initials were selected and copied as well.